src/codeu/chat/client/simplegui/SignInPanel.java

```diff
@@ -0,0 +1,59 @@
+package codeu.chat.client.simplegui;
+
+/**
+ * Created by Suveena on 5/18/17.
+ */
+
+import javax.swing.*;
+import java.awt.*;
+
+/*import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+
+import codeu.chat.client.ClientContext;
+import codeu.chat.common.User;*/
+
+@SuppressWarnings("serial")
+public final class SignInPanel extends JPanel {
+
+    public SignInPanel() {
+        super(new GridBagLayout());
+        initialize();
+    }
+
+    private void initialize() {
+
+        // This panel contains from top to bottom; a user message, Username field, Password field, and button
+
+        // Set layout within panel
+        JPanel InnerLayout = new JPanel();
```

Comment:
nit (style): please follow same semantics for variable naming: lowerCamelCase and use UpperCamelCase for class names only